pallas-primitives/src/alonzo/model.rs

```diff
@@ -697,7 +589,7 @@ mod tests {
             let bytes = hex::decode(header_str).unwrap_or_else(|_| panic!("bad header file {idx}"));
 
             let header: Header = minicbor::decode(&bytes[..])
-                .unwrap_or_else(|_| panic!("error decoding cbor for file {idx}"));
+                .unwrap();//_or_else(|_| panic!("error decoding cbor for file {idx}"));
```

Comment:
why remove the `_or_else`?

Reply:
It might have been a remnant of a debugging phase... Restored and tested.